fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexFunctionHelper.java

```diff
@@ -110,6 +112,25 @@ private IndexFunctionHelper() {
                 .min(Comparator.comparing(i -> i.state.index.getColumnSize()));
     }
 
+    public static Optional<Pair<IndexAggregateFunction, IndexMaintainer>> enumerateIndexMaintainersForAggregateFunctionCall(@Nonnull FDBRecordStore store,
```

Comment:
Javadoc (esp. since the min() at line 128 may have some assumptions)?

Reply:
Will add javadoc.